all-the-icons.el

```diff
@@ -878,12 +882,16 @@ Note: You want chevron, please use `all-the-icons-icon-for-dir-with-chevron'."
   "Get the formatted icon for FILE.
 ARG-OVERRIDES should be a plist containining `:height',
 `:v-adjust' or `:face' properties like in the normal icon
-inserting functions."
-  (let* ((ext (file-name-extension file))
-         (icon (or (and ext
-                        (cdr (assoc (downcase ext)
-                                    all-the-icons-extension-icon-alist)))
-                   (all-the-icons-match-to-alist file all-the-icons-regexp-icon-alist)))
+inserting functions.
+It will search for icons in the following order:
+`all-the-icons-regexp-icon-alist'
+`all-the-icons-extension-icon-alist'
+If none is found it will use `all-the-icons-default-icon'."
+  (let* ((ext? (file-name-extension file))
```

Comment:
Since
```lisp
(or (and ext
                        (cdr (assoc (downcase ext)
                                    all-the-icons-extension-icon-alist)))
```
was a little bit too long, I opted to create this new var called `ext?` and check its existence and downcasing it when defining `ext` var.